sftp-config.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
@jigkoxsee  Could you check this file  ? I not sure what is this environment variable.